app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/ContributorsViewModel.java

```diff
@@ -16,14 +16,29 @@
 import javax.inject.Inject;
 
 import io.github.droidkaigi.confsched2017.BR;
+import io.github.droidkaigi.confsched2017.R;
+import io.github.droidkaigi.confsched2017.di.scope.FragmentScope;
 import io.github.droidkaigi.confsched2017.repository.contributors.ContributorsRepository;
-import io.reactivex.Single;
+import io.github.droidkaigi.confsched2017.view.helper.ResourceResolver;
 import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
 
+@FragmentScope
 public final class ContributorsViewModel extends BaseObservable implements ViewModel, ContributorViewModel.Callback {
 
+    public static final String TAG = ContributorsViewModel.class.getSimpleName();
+
+    private final ResourceResolver resourceResolver;
+
+    private final ToolbarViewModel toolbarViewModel;
```

Comment:
But `ObservableList<ContributorViewModel> viewModels` has already been nested.
So I feed it's not big problem. It's just question!